### ISSUE TITLE
Retraining model from sleap 1.4 in 1.6, no longer able to find any instances

### ISSUE THREAD

**Issue by aavanderleij** [talmolab/sleap#2640]
## Bug description

I have a labeled training set ( n=1250) I that i used to train a model in sleap 1.4. I wanted to add some more labels  so it will perform better on newer videos as they have a slightly different lighting and placement of objects. However non of the new models are able to find any instances. I have tried a lot of different settings but all seem to fail. During training the loss goes down on the first 10 or so baches but then stays the same. Recall and precision are both 0. I had no issues training models before when using 1.4. I tried to retrain a model from scratch by starting starting a fresh project and relabeling + train just 1 video but still the same result (0 recall).

When running label QC it tells me over 94% of my labels are flagged, most with "Unusual Visibility" but I don't see any issues with the labels. Its a light brown bird on a sandy surface but it is very clearly visible.

I have tried with the same settings as before (top-down/multi animal) and tried single animal models. I also tested several different strides, augmentations, input scaling but the result is always 0 recall.

## Expected behaviour

The same or similar training results as with the previous versions.

## Actual behaviour

No instances are found after training with sleap 1.6.0, 0 recall during training.

## Screenshots

<img width="1908" height="995" alt="Image" src="https://github.com/user-attachments/assets/c222f90a-e5e6-4b80-aeaa-059e1638fbab" />

</details>

<details><summary>Logs</summary>

[training_log.csv](https://github.com/user-attachments/files/25795038/training_log.csv)

```
# paste relevant logs here, if any
```

</details>

**Comment by gitttt-1234** [comment]
Hi @aavanderleij !

We just released sleap-nn v0.1.1 with a few bug fixes, could you upgrade to the latest version and try training again? If you installed SLEAP via `uv tool install`, you can run to install the latest version:
```
uv tool upgrade sleap --upgrade-package sleap-nn
```
You can verify the installed version by running  `sleap doctor`.

Let us know if you have any questions!

Thanks,

Divya

**Comment by aavanderleij** [comment]
Hi @gitttt-1234 ,

Thank you for your quick response!

I have upgraded SLEAP to 1.6.2 (I was at 1.6.1 before) with sleap-nn at 0.1.2 and re-trained a model but the issues persist, I don't see any changes in the results.

Are there any other steps I can take? 

With kind regards,

Antsje

**Comment by gitttt-1234** [comment]
Hi @aavanderleij !

Could you please share the configs and the slp file [here](https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform) to help us debug better?

Thanks,

Divya

**Comment by aavanderleij** [comment]
Hi @gitttt-1234 ,

Just wanted to let you know I send the files.
Let me know if you need anything else.

Cheers,
Antsje

**Comment by aavanderleij** [comment]
Hi @gitttt-1234,
I was wondering if you had a chance to look at my issue yet.
i've had a look at the new releases but I did not see any fixes for 0 recall problem. Should I update test again or downgrade back to 1.4?
Is it the videos that cause this and is there anything I can do to prevent this from happening?

Cheers,
Antsje

**Comment by gitttt-1234** [comment]
Hi @aavanderleij ! 

Apologies for the delay :/!!!!

We're testing your data by training a new model on our side, will update here once we have the results! 

Thanks,

Divya

**Comment by gitttt-1234** [comment]
Hi @aavanderleij !

I was able to get good predictions on your data with a few modifications to the training pipeline:

- **Topdown model works better**: Since your instance is small relative to the frame, a top-down model performs better, even with only a single instance per frame. This approach first detects the centroid of each instance (centroid model), then crops the image around it and feeds the crop to the pose estimation model (centered-instance model).
- **Set Max Instances to 1**: Since you know there's only one instance per frame, setting this during inference helps avoid false positives (if any).
- **New Config Helper webapp**: We have a new [Config Helper](https://nn.sleap.ai/latest/configuration/config-picker/) that recommends the best configuration for your data and lets you visualize key settings.

Apologies for the delay!! 

I'm attaching the configs here, let us know if it works!!

[centered_instance_config.yaml](https://github.com/user-attachments/files/28156665/centered_instance_config.yaml)

[centroid_config.yaml](https://github.com/user-attachments/files/28156666/centroid_config.yaml)

Thanks,

Divya

**Comment by gitttt-1234** [comment]
Closing this issue, feel free to open a new issue if you have any questions!
